### PR TITLE
AAP 26574 - expose markdown content

### PR DESCRIPTION
**Pull request by SiddharthRajaraman**

### CODE REVIEW

File: src/features/lightspeed/lightspeedUser.ts [hunk 1]
```diff
@@ -300,6 +355,61 @@ export class LightspeedUser {
...
+        const userRole =
+          sessionInfo.userInfo?.role !== undefined
+            ? sessionInfo.userInfo?.role
+            : "";
+        content = `
+          <p><strong>Logged in as</strong>: ${userName}</p>
+          <p><strong>User Type</strong>: ${userType}</p>
+          ${userRole ? "Role: " + userRole : ""}
+        `;
+      } else {
```
Comment (goneri) [inline src/features/lightspeed/lightspeedUser.ts:405]: I would initialize `content` as `content = "undefined";` and drop this else block.

File: src/features/lightspeed/lightspeedUser.ts [hunk 2]
```diff
@@ -136,6 +140,47 @@ export class LightspeedUser {
...
+        `${getBaseUri(this._settingsManager)}${LIGHTSPEED_MARKDOWN_ME_AUTH_URL}`,
+        {
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        },
+      );
+
+      const markdownData = marked.parse(data.content) as string;
+      console.log("markdownData");
```
Comment (TamiTakamiya) [inline src/features/lightspeed/lightspeedUser.ts:159]: Are these two lines are for debug only?

Comment (SiddharthRajaraman) [reply src/features/lightspeed/lightspeedUser.ts:159]: yep... forgot to delete 🤦.

**Comment by TamiTakamiya** [comment]
In `lightspeedUser.ts`, there are two places where `this._userDetails` is reset to `undefined`. I think `this._markdownUserDetails` also needs to be reset to `undefined` there.

**Comment by SiddharthRajaraman** [comment]
> In `lightspeedUser.ts`, there are two places where `this._userDetails` is reset to `undefined`. I think `this._markdownUserDetails` also needs to be reset to `undefined` there.

Would that be lines 326 and 332?

**Review by TamiTakamiya — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**